### ISSUE TITLE
Auto mark zero dollar referrals as paid

### ISSUE THREAD

**Issue by cklosowski** [AffiliateWP/AffiliateWP#1455]
As a use case, I'm using myself as an affiliate to track the effectiveness of cross promotion between my own sites, and I have a 0% referral rate for my own affiliate account, but I have to mark them as paid. Would be nice to have an option (even via a filter) to auto mark $0 referrals as paid.

**Comment by pippinsplugins** [comment]
Makes sense to me. Perhaps the better option, however, would be to combine this with #794

**Comment by cklosowski** [comment]
Kind of:
```
Affiliate 1 sends customer = gets $30 credit
Affiliate 2 sends customer = gets $30 credit
No Affiliate sends customer (direct site visit) = Special Affiliate called Direct gets $50 credit
```

Only difference is that mine is a specific affiliate, not 'no affiliate'. I want to track specific links, not all direct links.

**Comment by pippinsplugins** [comment]
Ah, right. Thanks for the clarification.

**Comment by worldtravlr** [comment]
@cklosowski not sure if this would work, but if you are using a specific affiliate account/link and wanted to make a special sales/cart/checkout/order-completed page for only yourself to link to, then you could maybe use the Affiliate Conversion Script - http://docs.affiliatewp.com/article/77-affiliate-conversion-script - to set the referral to $0 and status to paid.
